### PR TITLE
[Core][Channel] Add contact phone number to channel

### PR DESCRIPTION
**Pull request by twojtylak**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | mentioned in #11886
| License         | MIT

Add channel contact phone number

Duplicate of #11943 (Fucked up the rebase process 😭 )

### CODE REVIEW

**Review by AdamKasp — APPROVED** [review]

**Review by lchrusciel — COMMENTED** [review]
Hey @twojtylak,

thanks for your proposal. It could really land into the core. However, I'm not sure if it is used anywhere. Can you customize the checkout template to make usage of this new field?
image

**Comment by twojtylak** [comment]
Hey @lchrusciel,

I adjusted the `_support.html.twig` template. The 'need_assistance' block is now only displayed if the channel has a contact phone number. I also adjusted the fixtures so that something is displayed here after loading them.

File: src/Sylius/Bundle/CoreBundle/Form/Extension/ChannelTypeExtension.php [hunk 1]
```diff
@@ -75,6 +76,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label' => 'sylius.form.channel.contact_email',
                 'required' => false,
             ])
+            ->add('contactPhoneNumber', TextType::class, [
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Form/Extension/ChannelTypeExtension.php:79]: Could be improved to:
```suggestion
            ->add('contactPhoneNumber', TelType::class, [
```
Ref. https://symfony.com/doc/current/reference/forms/types/tel.html

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @twojtylak! :tada:

**Pull request merged**